test/parallel/test-console-assing-undefined.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// Should be above require, because code in require read console
+// what we are trying to avoid
+// set should be earlier than get
+
+global.console = undefined;
+
+// Initially, the `console` variable is `undefined`, since console will be
+// lazily loaded in the getter.
+
```

Comment:
@Wandalen almost there :) The `common` module should be the first one that a test `require`s. Please see our [guide on writing tests](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md).

> Do you participate in OdessaJS this year?

Yep.